jupyros/ros2/Subscription.py

```diff
@@ -0,0 +1,187 @@
+"""
+Subscription class for jupyter-ros2 Project
+
+Modified by: ldania (Luigi Dania)
+Date: 22-July-2022
+
+
+Original Author: zmk5 (Zahi Kakish)
+
+
+"""
+from typing import TypeVar
+from typing import Callable
+import threading
+import functools
+import ipywidgets as widgets
+
+try:
+    import rclpy
+    from rclpy.node import Node
+except ModuleNotFoundError:
+    print("The rclpy package is not found in your $PYTHONPATH. " +
+          "Subscribe and publish are not going to work.")
+    print("Do you need to activate your ros2 environment?")
+
+
+# Used for documentation purposes only
+MsgType = TypeVar('MsgType')
+
+
+class Subscription():
```

Comment:
This should be renamed to `Subscriber` to keep it consistent with `Publisher` and also because in `__init__.py` it is mentioned as `Subscriber`.